src/delta5node/delta5node.ino

```diff
@@ -254,7 +261,11 @@ void setRxModule(int frequency) {
 
 // Read the RSSI value for the current channel
 int rssiRead() {
-	return analogRead(0);
+	#ifdef LEONARDO
```

Comment:
instead of the define here, can you add a variable for the pin with the other variables?

Reply:
ok